tests/integration/test_podman_compose_network_scoped_aliases.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
It is fine to not have integration test, if the unit tests cover the code like in this case. Thanks for adding it anyway.